.github/workflows/doc.yml

```
@@ -0,0 +1,26 @@
+name: Build LaTeX documentation
+on: [push]
+jobs:
+  build_latex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up repository
+        uses: actions/checkout@v2
```

There is version 3 now:
```suggestion
        uses: actions/checkout@v3
```